scripts/genebuild/create_annotation_configs.pl

```diff
@@ -594,7 +594,7 @@ sub clade_settings {
     'rodentia' => {
       'repbase_library'    => 'rodentia',
       'repbase_logic_name' => 'rodentia',
-      'uniprot_set'        => 'mammals_basic',
+      'uniprot_set'        => 'rodentia_basic',
```

Comment:
The uniprot_set is used elsewhere s parameter, e.g. for layering it chooses the correct set of layers from https://github.com/Ensembl/ensembl-analysis/blob/main/modules/Bio/EnsEMBL/Analysis/Hive/Config/LayerAnnotationStatic.pm based on the "uniprot_set". An entry for "rodentia_basic" does not exist in the layer static, so you would need to introduce. It might be simpler to introduce a new parameter for the sanity checks, e.g. "sanity_set"

Reply:
Looks good! I think this makes it simpler to introduce different sanity sets for other clades too, if we need them!